snpeff/5.1/Dockerfile

```diff
@@ -0,0 +1,70 @@
+ARG SNPEFF_VER="5.1d"
+ARG SNPSIFT_VER="5.1"
+
+FROM ubuntu:focal as app
+
+ARG SNPEFF_VER
+ARG SNPSIFT_VER
+
+# Metadata
+LABEL base.image="ubuntu:focal"
+LABEL dockerfile.version="1"
+LABEL software="SnpEff & SnpSift"
+LABEL snpeff.software.version=$SNPEFF_VER
+LABEL snpsift.software.version=$SNPSIFT_VER
+LABEL description.SnpEff="Genetic variant annotation and effect prediction toolbox."
+LABEL description.SnpSift="Used after SnpEff annotation to filter and manipulate annotated files."
+LABEL website="https://pcingola.github.io/SnpEff"
+LABEL license="https://github.com/pcingola/SnpEff/blob/master/LICENSE.md"
+LABEL maintainer="Tom Iverson"
+LABEL maintainer.email="tiverson@utah.gov"
+
+# Install open using apt 
+RUN apt-get update && apt-get install -y software-properties-common &&\
+  apt-get update && apt-get install -y \
+  openjdk-11-jre \
+  ant \
+  maven \
+  wget \
+  unzip
+
+# Use wget to get the source code for SnpEff and SnpSift from GitHub.
+RUN wget https://github.com/pcingola/SnpEff/archive/refs/tags/v${SNPEFF_VER}.tar.gz && \
+  tar -xvf v${SNPEFF_VER}.tar.gz && \
+  rm v${SNPEFF_VER}.tar.gz && \ 
+  wget https://github.com/pcingola/SnpSift/archive/refs/tags/v${SNPSIFT_VER}.tar.gz && \
+  tar -xvf v${SNPSIFT_VER}.tar.gz && \
+  rm v${SNPSIFT_VER}.tar.gz
+
+# Build libraries from the source code using Maven.
+RUN cd SnpEff-${SNPEFF_VER}/lib && \
+  mvn install:install-file -Dfile=antlr-4.5.1-complete.jar -DgroupId=org.antlr -DartifactId=antlr -Dversion=4.5.1 -Dpackaging=jar && \
+  mvn install:install-file -Dfile=biojava3-core-3.0.7.jar -DgroupId=org.biojava -DartifactId=biojava3-core -Dversion=3.0.7 -Dpackaging=jar && \
+  mvn install:install-file -Dfile=biojava3-structure-3.0.7.jar -DgroupId=org.biojava -DartifactId=biojava3-structure -Dversion=3.0.7 -Dpackaging=jar
+
+# Once libraries are installed, use make.sh to build the code.
+COPY make.sh .
```

Comment:
Instead of using a make.sh file that you've included, could you use the make.sh that's part of their github repo instead of this one? It'd make things easier for the next snpeff update.

Although... why are you using make.sh when you're installing the dependencies in lines 41-43.

Reply:
I've added the code to build the snpeff/snpsift jar executables instead of copying the make.sh file, as well as removed the galaxy.sh and test-snpeff.yml files. Hopefully, it should be good to go now.